app/helpers/breadcrumbs_helper.rb

```diff
@@ -0,0 +1,9 @@
+module BreadcrumbsHelper
+  def render_breadcrumbs(breadcrumbs_hash)
+    @breadcrumbs ||= []
```

Comment:
no need
`#each` method comes from Enumerable, so it's applicable for hashes too, which you'r using below